pom.xml

```diff
@@ -57,39 +57,98 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bnd-process</id>
+                        <goals>
+                            <goal>bnd-process</goal>
+                        </goals>
+                        <configuration>
+                            <bnd>
+-noextraheader:true
+-snapshot: SNAPSHOT
+Sling-Namespaces: slingevent=http://sling.apache.org/jcr/event/1.0
+Sling-Nodetypes: SLING-INF/nodetypes/event.cnd
+Import-Package: \
+    javax.servlet.http;resolution:=optional,\
+    org.apache.felix.inventory;resolution:=optional,\
+    javax.servlet;resolution:=optional,\
+    <!-- For build compatibility with Java 11 -->\
+    javax.annotation;version="[0.0,2)",\
+    *
+Export-Package: \
+    org.apache.sling.event.jobs,\
+    org.apache.sling.event.jobs.consumer,\
+    org.apache.sling.event.jobs.jmx
+
+-conditionalpackage: \
+    org.apache.jackrabbit.util,\
+    org.apache.sling.commons.osgi,\
+    org.apache.sling.event.api
+-includeresource: \
+    target/dependency/org.apache.sling.event.api.jar,\
+    @target/dependency/quartz.jar!/org/quartz/CronExpression.class,\
+    @target/dependency/quartz.jar!/org/quartz/ValueSet.class,\
+    @target/dependency/org.apache.sling.commons.osgi.jar!/org/apache/sling/commons/osgi/PropertiesUtil.class,\
+    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/ISO9075.class,\
+    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/ISO8601.class,\
+    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/XMLChar.class
+Bundle-Classpath: ., org.apache.sling.event.api.jar
+                            </bnd>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-baseline-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Instead of relying on maven-dependency-plugin I would rather rely on bnd for adding the third party dependencies (e.g. via https://stackoverflow.com/questions/58718160/how-to-embed-a-maven-dependency-with-bnd-maven-plugin)

Reply:
this one I'm not sure what you mean exactly. The `bnd.bnd` already contains an `-includeresource` section, and that one is referring to jar files that I gues the `maven-dependency-plugin` plugin deposits into `target/dependencies`. How are you suggesting to simplify this?

Reply:
ok, I guess I found a way